clean-lumber.lic

```diff
@@ -24,7 +24,7 @@ class CleanLumber
   end
 
   def process_size(args, size)
-    while bput("get #{args.type} #{size} from my #{args.source}", 'You get', 'You carefully remove', 'What were you') != 'What were you'
```

Comment:
Our style standard has been the order you saw. The order shouldn't matter, so I'm a bit concerned something else could be going on.